src/views/core/circles/ViewMembers.vue

```diff
@@ -131,7 +131,7 @@ export default {
       })
     },
     deleteMember (membership) {
-      this.axios.delete(this.services['oms-core-elixir'] + '/circles/' + this.$route.params.id + '/members/' + membership.id).then(() => {
+      this.axios.delete(this.services['corre'] + '/circles/' + this.$route.params.id + '/members/' + membership.id).then(() => {
```

Comment:
There's a typo here, you put `corre` instead of `core`

Reply:
ah yeah, my MBP's fault, fixed now.